### PR TITLE
Add Logging 6.1 jobs

### PR DESCRIPTION
**Pull request by anpingli**

### CODE REVIEW

**Comment by anpingli** [comment]
/pj-rehearse periodic-ci-openshift-openshift-tests-private-release-4.18-amd64-nightly-openshift-logging-6.1-gcp-ipi-sdn-fips-f7

**Comment by anpingli** [comment]
/pj-rehearse  periodic-ci-openshift-openshift-tests-private-release-4.15-multi-stable-openshift-logging-6.1-azure-ipi-oidc-mixarch-f7

**Comment by anpingli** [comment]
/pj-rehearse periodic-ci-openshift-openshift-tests-private-release-4.15-multi-stable-openshift-logging-6.1-azure-ipi-oidc-mixarch-f7

**Comment by anpingli** [comment]
/pj-rehearse periodic-ci-openshift-openshift-tests-private-release-4.18-amd64-nightly-openshift-logging-6.1-gcp-ipi-sdn-fips-f7

**Comment by anpingli** [comment]
@QiaolingTang  PTAL

**Comment by anpingli** [comment]
/pj-rehearse periodic-ci-openshift-openshift-tests-private-release-4.16-amd64-stable-openshift-logging-6.1-gcp-ipi-proxy-oidc-amd-f7-logging

**Comment by anpingli** [comment]
/pj-rehearse periodic-ci-openshift-openshift-tests-private-release-4.17-multi-stable-openshift-logging-6.1-aws-ipi-proxy-sts-mixarch-f7-logging

**Comment by anpingli** [comment]
/pj-rehearse periodic-ci-openshift-openshift-tests-private-release-4.18-amd64-nightly-openshift-logging-6.1-gcp-ipi-fips-f7-logging

**Comment by anpingli** [comment]
/pj-rehearse periodic-ci-openshift-openshift-tests-private-release-4.15-multi-stable-openshift-logging-6.1-azure-ipi-oidc-mixarch-f7-logging

**Comment by anpingli** [comment]
@liangxia  PTAL

File: ci-operator/config/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.15__multi-stable-openshift-logging-6.1.yaml [hunk 1]
```diff
@@ -0,0 +1,89 @@
...
+      version: "4.15"
+resources:
+  '*':
+    limits:
+      memory: 8Gi
+    requests:
+      cpu: "1"
+      memory: 1Gi
+tests:
+- as: azure-ipi-oidc-mixarch-f7-logging
```
Comment (liangxia) [inline ci-operator/config/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.15__multi-stable-openshift-logging-6.1.yaml:61]: 4.15 release every 3 weeks. Perhaps we can use lower test frequency in 4.15 ?

Comment (anpingli) [reply ci-operator/config/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.15__multi-stable-openshift-logging-6.1.yaml:61]: @liangxia  It is Openshift Logging 6.1 test, not for 4.15. Logging will be release at Nov 13. we can use  lower test frequency  after that.

Comment (liangxia) [reply ci-operator/config/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.15__multi-stable-openshift-logging-6.1.yaml:61]: The other logging version 5.8/5.9/6.0 with OCP 4.15 are all in `f7`
```
openshift-openshift-tests-private-release-4.15__amd64-nightly-openshift-logging-5.9.yaml:- as: azure-ipi-proxy-workers-rhcos-rhel8-f7-logging
openshift-openshift-tests-private-release-4.15__amd64-nightly-openshift-logging-5.9.yaml:- as: aws-ipi-f7-disruptive-logging
openshift-openshift-tests-private-release-4.15__multi-nightly-openshift-logging-5.8.yaml:- as: aws-ipi-ovn-ipsec-arm-mixarch-f7-logging
openshift-openshift-tests-private-release-4.15__multi-stable-openshift-logging-6.0.yaml:- as: azure-ipi-oidc-mixarch-f7-logging
```

Comment (anpingli) [reply ci-operator/config/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.15__multi-stable-openshift-logging-6.1.yaml:61]: Logging z-stream will be released every sprint (3 week).   Currently, we support 5.8,5.9,6.0. 6.1 is the next major release.

Comment (anpingli) [reply ci-operator/config/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.15__multi-stable-openshift-logging-6.1.yaml:61]: There are about 4*4 =16 jobs for Logging ever week.

Comment (liangxia) [reply ci-operator/config/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.15__multi-stable-openshift-logging-6.1.yaml:61]: For logging 6.1, I'm fine with testing it weekly (f7).  
I don't understand for the others.  E.g, ocp 4.15+logging 5.9, both ocp 4.15 and logging 5.9 release every 3 weeks, why it's required to test every weeks ?
CC @jianlinliu as this is cloud cost related.

Comment (anpingli) [reply ci-operator/config/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.15__multi-stable-openshift-logging-6.1.yaml:61]: @liangxia  Can you merge this？ I can decrease the frequency to bi-week for Logging 5.8,5.9 and 6.1 in another PR.

Comment (liangxia) [reply ci-operator/config/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.15__multi-stable-openshift-logging-6.1.yaml:61]: Ack

**Pull request merged**